CHANGELOG.md

```diff
@@ -105,7 +108,7 @@ All notable changes to Pixano will be documented in this file.
 ### Changed
 
 - Separate image tools and annotation tools in Pixano Annotator
-- Refactor Pixano Apps components
+- Refactor Pixano apps components
```

Comment:
apps or app? (don't know if it's for the two or the new merged app)

Reply:
This is a changelog of an older version with the two apps, this commit is just fixing the case for consistency